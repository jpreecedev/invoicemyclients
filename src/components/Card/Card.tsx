import React from 'react';

interface CardProps {
	title?: string;
}

const Card: React.FC<CardProps> = ({ title = '', children }) => {
	return (
		<div className="card shadow-light-lg card-border border-success mb-5">
			<div className="card-body">
				{title && <h2 className="card-title text-body">{title}</h2>}
				{children}
			</div>
		</div>
	);
};

export { Card };
