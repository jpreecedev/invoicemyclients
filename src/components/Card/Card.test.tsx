import React from 'react'
import { render } from '@testing-library/react'
import { Card } from './Card'

describe('<Card /> tests', () => {
  it('should match snapshot', () => {
    expect(render(<Card />).container).toMatchSnapshot()
  })
})
