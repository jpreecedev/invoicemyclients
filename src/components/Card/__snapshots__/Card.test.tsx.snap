// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> tests should match snapshot 1`] = `
<div>
  <div
    class="card shadow-light-lg card-border border-success mb-5"
  >
    <div
      class="card-body"
    >
      
    </div>
  </div>
</div>
`;
